src/cascadia/TerminalSettingsEditor/GlobalSettingsViewModel.h

```diff
@@ -0,0 +1,74 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+#pragma once
+
+#include "GlobalSettingsViewModel.g.h"
+#include "Utils.h"
+#include "ViewModelHelpers.h"
+#include "..\TerminalSettingsModel\MTSMSettings.h"
+#include "Utils.h"
+
+namespace winrt::Microsoft::Terminal::Settings::Editor::implementation
+{
+    struct GlobalSettingsViewModel : GlobalSettingsViewModelT<GlobalSettingsViewModel>, ViewModelHelper<GlobalSettingsViewModel>
+    {
+    public:
+        GlobalSettingsViewModel(const Model::GlobalAppSettings& globalSettings, const Model::CascadiaSettings& appSettings);
+
+        void SetInvertedDisableAnimationsValue(bool invertedDisableAnimationsValue);
+
+        _GET_SETTING_FUNC(_globals, DefaultProfile);
+        _SET_SETTING_FUNC(_globals, DefaultProfile);
+        _CLEAR_SETTING_FUNC(_globals, Language);
+
+        // Launch

```

Comment:
(note to self) don't organize these by page. Organize them like this:
- custom getters/setters
- GETSET_BINDABLE_ENUM_SETTING
- X-Macro